how-we-work/index.html

```diff
@@ -0,0 +1,116 @@
+---
+layout: blank
+title: How we work
+description: We use the SCRUM methodology and we’ll be happy to introduce its principles to you.
+keywords: methodology, scrum, agile, flexible contract
+---
+<div class="content-block">
+  <article class="v-how-we-work typeset" itemtype="http://schema.org/Product" itemscope>
+    <meta itemprop="name" content="custom-tailored software">
+    <meta itemprop="category" content="software development">
+    <span itemprop="audience" itemtype="https://schema.org/Audience" itemscope class="hidden">
+      <meta itemprop="audienceType" content="anyone">
+    </span>
+    <span itemprop="brand" itemscope itemtype="http://schema.org/Organization">
+      <meta itemprop="name" content="{{ site.company.official_name }}">
+    </span>
+    <span itemprop="manufacturer" itemscope itemtype="http://schema.org/Organization">
+      <meta itemprop="name" content="{{ site.company.official_name }}">
+    </span>
+    <section class="hero hero--nobottom">
+      <h1 class="hero__headline">How we work</h1>
+    </section>
+    <main class="article-typeset">
+      <section class="text-center">
+        <img src="/assets/img/working.svg" alt="">
+        <span class="section__main-title">First, we’d like to explain how we work.</span>
+        <p>We use the <a href="https://cs.wikipedia.org/wiki/Scrum" class="anchor anchor--emphasized" target="_blank" rel="noopener">SCRUM</a> methodology and we’ll be happy to introduce its principles to you.</p>
+      </section>
+      <section class="text-center">
+        <span class="section__main-count">1</span>
+        <span class="section__main-title">We’ll agree on your team’s lineup.</span>
+        <p>Together we’ll decide on your team’s optimal composition.</p>
```

Comment:
[nice-to-have] Seems like the `’` character should rather read `'`. Multiple occurences.